cupy/sparse/__init__.py

```diff
@@ -16,6 +16,10 @@
 from cupyx.scipy.sparse.construct import random  # NOQA
 from cupyx.scipy.sparse.construct import spdiags  # NOQA
 
+from cupyx.scipy.sparse.construct import vstack
+from cupyx.scipy.sparse.construct import hstack
+from cupyx.scipy.sparse.construct import bmat
```

Comment:
These three lines will need  `# NOQA` as in the imports above to avoid flake8 errors on CI:
https://travis-ci.org/cupy/cupy/jobs/614732943#L315-L317

see also the other minor formatting issue raised in that same log